util.go

```diff
@@ -104,6 +104,11 @@ func (c *Config) updateSnapshot(snapshotName string, snapshot string) error {
 		return err
 	}
 
+	if !c.failOnUpdate {
+		//TODO: should a warning still be printed here?
```

Comment:
@byang223 Are you expecting updating to occur completely silently if this option is enabled?

Reply:
I don't want it to fail go test when this option is enabled. It could be nice to log a diff between old/new when snapshot updated and log "new snapshot created" when new snapshots are created

Reply:
Sounds like a good idea but I'm going to make a separate issue for it (as it's going to take some refactoring)

Reply:
Yep totally fine, thanks for making this change!